src/thenCatchP.js

```diff
@@ -0,0 +1,29 @@
+import { invoker } from 'ramda';
+
+/**
+ * Composable shortcut for `Promise.then` that allows for success and failure call backs.
+ * The thenCatchP function returns a Promise. It takes three arguments: a callback function for the success of the Promise,
+ * a callback function for the failure of the Promise, and the promise instance itself.
+ *
+ * @func thenP
+ * @memberOf RA
+ * @aliases then
```

Comment:
```suggestion
```

Reply:
Sorry for leaving this in, thanks for finding it.